### DISCUSSION TITLE
Does ruff respect XDG_CONFIG_HOME environment var?

### DISCUSSION THREAD

**Discussion by bavalpey** [astral-sh/ruff#10157] [Q&A]
I saw in #215 some discussion of support for $XDG_CONFIG_HOME. However, being honest, it's unclear to me if this was eventually supported. If I have set $XDG_CONFIG_HOME to some arbitrary location, is ruff going to respect that?

If so, should probably update the documentation.
Would also be nice to see a RUFF_CONFIG_GLOBAL environment variable sort akin to git's GIT_CONFIG_GLOBAL.

**Comment by charliermarsh** [comment]
Yeah, we use the [`dirs`](https://crates.io/crates/dirs) crate, so whatever they return for `dirs::config_dir` (see the linked README).
